docs/source/interfaces/muon/muon_corrections_tab.rst

```diff
@@ -0,0 +1,73 @@
+.. _muon_corrections_tab-ref:
+
+Corrections Tab
+---------------
+
+.. image::  ../../images/muon_interface_tab_corrections.png
+   :align: right
+   :height: 400px
+
+The corrections tab is used for correcting the data to be analysed.
+There are two types of corrections; dead time corrections, and background corrections.
+
+**Runs** Allows you to select the runs to display correction data for.
+
+Dead Time Corrections
+^^^^^^^^^^^^^^^^^^^^^
+
+**Dead Time for all runs** This allows you to select the method to use for dead time corrections for all runs. There are four deadtime correction options:
+
+* ``From data file``: The deadtime table will be taken from the relevant data file
+* ``From table workspace``: A previously loaded deadtime table may be selected which will then be used in all group and pair calculations.
+* ``From other file``: A file may be selected to load a deadtime table from this will then be used in all group and pair calculations.
+* ``None``: No deadtime corrections should be applied.
+
+**Dead Time stats** Displays the minimum, maximum and average deadtime correction for the currently selected runs.
+
+Background Corrections
+^^^^^^^^^^^^^^^^^^^^^^
+
+**Background** There are three background correction options:
+
+* ``None``: No background correction should be applied.
+* ``Auto``: The background is automatically calculated by performing a fit on the loaded data before being subtracted from the data. The diagram below describes this process.
```

Comment:
The placement of the diagram is more to the side than below. Perhaps this should be reworded?